services/121-service/src/payments/transactions/transactions.controller.ts

```diff
@@ -31,7 +31,9 @@ export class TransactionsController {
 
   // TODO: Remove this endpoint when the old portal has been deprecated.
```

Comment:
This happened, so remove this code and code it depends on that is not used elsewhere.

Reply:
Is already done in another PR